docs/uc.md

```diff
@@ -0,0 +1,60 @@
+# Using dbt-databricks with Unity Catalog
+
+[Unity Catalog](https://www.databricks.com/product/unity-catalog) provides a SQL interface to manage tables, views and more across Databricks workspaces using standard SQL. dbt-databricks supports 
+
+# Benefits of Unity Catalog for analytics engineers
+
+We strongly recommend using Unity Catalog for all dbt projects on Databricks. Unity Catalog brings two important benefits to analytics engineers:
+
+1. Unity Catalog offers a three-level namespace: catalog > schema > table/view. Note that schema is a synonym for database in earlier releases of dbt-databricks. This lets you manage and isolate data and avoids polluting a single database with unrelated tables and views. For example, you can data between your finance and HR teams inside a production catalog called `prod` in two schemas, `finance` and `hr`:
```

Comment:
> you can data between ...

Do you mean `you can isolate date` or `you can separate data` or something like that?